src/features/collectables/TransferCompleteScreen.tsx

```diff
@@ -0,0 +1,204 @@
+import { useSpacing } from '../../theme/themeHooks'
+import useCollectables from '../../utils/useCollectables'
+import { RootState } from '../../store/rootReducer'
+
+LogBox.ignoreLogs([
+  'Non-serializable values were found in the navigation state',
+])
+
+type Route = RouteProp<CollectableStackParamList, 'TransferCompleteScreen'>
+
+const TransferCollectableScreen = () => {
+  const route = useRoute<Route>()
+  const navigation = useNavigation<CollectableNavigationProp>()
+  const COLLECTABLE_HEIGHT = ww
+  const backEdges = useMemo(() => ['top'] as Edge[], [])
+
+  const { t } = useTranslation()
+  const { collectable } = route.params
+  const { refresh } = useCollectables()
+  const solanaPayment = useSelector(
+    (reduxState: RootState) => reduxState.solana.payment,
+  )
+  const spacing = useSpacing()
+
+  const backgroundImageUri = useMemo(() => {
+    return collectable?.json?.image
+  }, [collectable.json])
+
+  const onReturn = useCallback(() => {
+    navigation.popToTop()
+  }, [navigation])
+
+  useEffect(() => {
+    return () => {
+      // Refresh collectables on return
+      refresh()
+    }
+  }, [refresh])
+
+  if (!collectable.json || !backgroundImageUri) {
+    return null
+  }
+
+  return (
+    <BackScreen
+      padding="none"
+      backgroundImageUri={backgroundImageUri}
+      edges={backEdges}
+      onClose={onReturn}
+      hideBack
+    >
+      <Animated.View entering={DelayedFadeIn} style={globalStyles.container}>
+        <Box
+          backgroundColor="transparent"
+          flex={1}
+          padding="m"
+          alignItems="center"
+          justifyContent="center"
+        >
+          <Box
+            flexGrow={1}
+            marginBottom="xl"
+            justifyContent="center"
+            alignItems="center"
+          >
+            {collectable.json && (
+              <Box
+                shadowColor="black"
+                shadowOpacity={0.4}
+                shadowOffset={{ width: 0, height: 10 }}
+                shadowRadius={10}
+                elevation={12}
+              >
+                <ImageBox
+                  marginTop="l"
+                  backgroundColor="black"
+                  height={COLLECTABLE_HEIGHT - spacing.xl * 5}
+                  width={COLLECTABLE_HEIGHT - spacing.xl * 5}
+                  source={{ uri: collectable.json.image, cache: 'force-cache' }}
+                  borderRadius="xxl"
+                />
+              </Box>
+            )}
+            {solanaPayment && !solanaPayment.error && !solanaPayment.loading && (
+              <Animated.View
+                style={{ alignItems: 'center' }}
+                entering={FadeIn}
+                exiting={FadeOut}
+              >
+                <Text variant="h2" color="white" marginTop="xl">
+                  {t('collectablesScreen.transferComplete')}
+                </Text>
+              </Animated.View>
+            )}
+
+            {solanaPayment && solanaPayment.error && (
```

Comment:
can always use optional chaining `{solanaPayment?.error && (`

Reply:
Always forget about this! Will do 🙏🏼